### ISSUE TITLE
Hash Mismatch while updating VS Code Insiders Version

### ISSUE THREAD

**Issue by jamestansx** [microsoft/vscode#112004]
Issue Type: <b>Bug</b>

Step to reproduce: 
1. Click 'Check for updates'
2.  A notification will prompted out stating that "Hash mismatch"

VS Code version: Code - Insiders 1.52.0-insider (5e350b1b79675cecdff224eb00f7bf62ae8789fc, 2020-12-04T10:15:27.849Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz (8 x 1190)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.81GB (2.97GB free)|
|Process Argv|--verbose --crash-reporter-id 68485aea-3c6b-4d06-8749-b90eb062e1ef|
|Screen Reader|no|
|VM|40%|
</details><details><summary>Extensions (28)</summary>

Extension|Author (truncated)|Version
---|---|---
codesnap|adp|1.2.0
arepl|alm|2.0.1
path-intellisense|chr|2.3.0
bracket-pair-colorizer-2|Coe|0.2.0
doxdocgen|csc|1.1.0
gitlens|eam|11.0.6
prettier-vscode|esb|5.8.0
vscode-test-explorer|hbe|2.19.3
output-colorizer|IBM|0.1.2
vsc-python-indent|Kev|1.12.0
vscode-python-test-adapter|lit|0.6.3
vscode-docker|ms-|1.8.1
vscode-ros|ms-|0.6.4
python|ms-|2020.12.401662388-dev
vscode-pylance|ms-|2020.12.0
jupyter|ms-|2020.11.399280825
remote-containers|ms-|0.152.1
remote-wsl|ms-|0.52.0
cpptools|ms-|1.1.3
autodocstring|njp|0.5.4
platformio-ide|pla|2.2.1
quicktype|qui|12.0.46
arduino-snippets|ron|1.0.2
partial-diff|ryu|1.4.1
indenticator|Sir|0.7.0
code-spell-checker|str|1.10.2
tabnine-vscode|Tab|3.1.16
vscode-icons|vsc|11.0.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
openlogontheside:30221882
python763:30178808
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
python504:30227505
vswsl492:30208929
wsl2promptcf:30219163
vstry244:30230484
pythonvsdeb440:30224570
unusedprompt:30219164
folderexplorercf:30219167
openfilemenu:30219168
pythonvsded773:30223139

```

</details>

**Comment by joaomoreno** [comment]
Yup, this was an oversight from last friday. We fixed it today (https://github.com/microsoft/vscode/commit/ea0b7fd29af51136688960c9ded4209964e3fe99) and are getting new builds running.
